aztk/spark/models/plugins/install/conda/configuration.py

```diff
@@ -0,0 +1,18 @@
+import os
+from aztk.models.plugins.plugin_configuration import PluginConfiguration, PluginPort, PluginTargetRole
+from aztk.models.plugins.plugin_file import PluginFile
+from aztk.utils import constants
+
+dir_path = os.path.dirname(os.path.realpath(__file__))
+
+def CondaPlugin(packages=None):
+    return PluginConfiguration(
+        name="conda",
+        target_role=PluginTargetRole.All,
+        execute="install.sh",
+        files=[
+            PluginFile("install.sh", os.path.join(dir_path, "..", "install.sh"))
+        ],
+        args=packages,
+        env=dict(COMMAND='conda install')
```

Comment:
Looks like the `-y` flag is necessary here to get around prompting. https://conda.io/docs/commands/conda-install.html

Reply:
WIll add the -y flag